spec/serializers/federation_entity_serializer_spec.rb

```diff
@@ -0,0 +1,20 @@
+describe FederationEntitySerializer do
+  class TestEntity < DiasporaFederation::Entity
+    property :test, :string
+  end
+
+  let(:object) { double }
+
+  def enable_test_entity_builder
+    allow(Diaspora::Federation::Mappings).to receive(:builder_for).with(object.class).and_return(:test_entity)
+    allow(Diaspora::Federation::Entities).to receive(:test_entity).with(object) {
+      TestEntity.new(test: "asdf")
+    }
+  end
+
+  before do
+    enable_test_entity_builder
```

Comment:
I think a small comment is better here (it's only one extra line and is simpler), instead of creating an extra method (which is 3 lines + one empty line, and is more complex, because I first see the method, and then need to check where it's used and see that it's the `before` step).